sopel/plugin.py

```diff
@@ -540,7 +544,7 @@ def add_attribute(function):
     return add_attribute
 
 
-def command(*command_list):
+def command(*command_list) -> typing.Callable:
```

Comment:
Just doing this once, but the same idea applies to all argument lists in this module.

```suggestion
def command(*command_list: str) -> typing.Callable:
```

(Don't like the name `command_list` and never have, but this isn't the time to change it.)